### ISSUE TITLE
Requiring portal.web doesn't work in clojurescript/node

### ISSUE THREAD

**Issue by farcaller** [gnl/playback#3]
`playback.core` requires `portal.web` if it's running under cljs: https://github.com/gnl/playback/blob/master/src/playback/core.cljc#L20. This immediately crashes nodejs runtime as it's missing the window global.

There needs to be some alternative that allows to use `portal.web` even in cljs.

**Comment by gnl** [comment]
Yup, it's not node-compatible yet, looking into it for the next release. The goal is to eventually support everything Portal supports, and node's the only one that's still missing.
